### ISSUE TITLE
Use external files as dictionaries

### ISSUE THREAD

**Issue by AlexanderZeilmann** [valentjn/vscode-ltex#118]
**Is your feature request related to a problem? Please describe.**
I am using other spell checkers besides `vscode-ltex` at the same time on the same document.
For this, I am maintaining several dictionaries in text files (one word per line).
One dictionary lists names of people which can be used for multiple languages, while other dictionaries list specific words,
which are only needed for one language.
I'd like to use these files as dictionaries which are considered by `vscode-ltex`.

Currently it looks as if I have to write the words from my dictionary directly into the `settings.json` file using one of
* ltex.dictionary
* ltex.workspaceDictionary
* ltex.workspaceFolderDictionary

This is undesirable because then I have to maintain several dictionaries for different spellcheckers.

**Describe the solution you'd like**

Three new settings:
* ltex.dictionaryFile
* ltex.workspaceDictionaryFile
* ltex.workspaceFolderDictionaryFile
which accept a JSON object with languages as keys, which in turn reference arrays of file names of dictionary files.
Something like this:

```json
"ltex.workspaceFolderDictionaryFile": {
        "en-US": [
                "./dictionaries/Mathematicians.txt",
                "./dictionaries/Mathematical_Terms.txt"
        ],
        "de-DE": [
                "./dictionaries/Mathematicians.txt",
                "./dictionaries/Mathematische_Ausdrücke.txt"
        ],
}
```
The dictionary files are loaded and their content appended to the corresponding entry in `"ltex.workspaceFolderDictionary"` (or `ltex.dictionary` or `ltex.workspaceDictionary`).

_This is only a quick idea for an API. I'm also happy with other implementations._

**Describe alternatives you've considered**
Somehow automatically copying the files from the dictionary file to the `settings.json` or maintaining the same words in two locations do not look like perfect options.

**Additional context**
Maybe this is already possible. But I haven't found it in the documentation.

**Comment by valentjn** [comment]
Thanks for the feature request. As I'm trying to avoid to add even more settings, the implemented solution is that entries that are in the regular dictionary settings and that start with a colon denote paths to external dictionary files (after stripping the colon, of course).

**Comment by AlexanderZeilmann** [comment]
Thank you so much for the fast implementation of this feature! Keep up your awesome work 😃

**Comment by valentjn** [comment]
No worries. Released in 7.1.0 with a fix in 7.1.1.

**Comment by stakaz** [comment]
Hello, would it also be possible to save the new files to the provided file instead of append them to the list? I like to have a global custom dictionary file for the whole system. So I would use a link to that file and this would add the new words globally for the whole system at once.
